src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -310,14 +309,20 @@ private static function getService()
 	/**
 	 * Return the ActivityPub profile of the given user
 	 *
-	 * @param integer $uid User ID
+	 * @param int $uid User ID
 	 * @return array with profile data
-	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 * @throws HTTPException\NotFoundException
+	 * @throws HTTPException\InternalServerErrorException
 	 */
-	public static function getProfile($uid)
+	public static function getProfile(int $uid): array
 	{
 		$owner = User::getOwnerDataById($uid);
 
+		if (!isset($owner['id'])) {
+			DI::logger()->error('Unable to find owner data for uid', ['uid' => $uid, 'callstack' => System::callstack()]);
```

Comment:
The `callstack` function only display the last 4 entries by default. When debugging it is helpftul to have some more entries.

Reply:
How many do you think I should set it to? Is there an infinite stack parameter value?

Reply:
I normally set it to 20.